src/GraphQL.NewtonsoftJson/HttpResponseStreamWriter.cs

```diff
@@ -9,7 +9,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace GraphQL.Http
+namespace GraphQL.NewtonsoftJson
```

Comment:
`HttpResponseStreamWriter` can be made `internal` like `JsonArrayPool`.

Reply:
It should be listed as internal now and the public api json has been updated to indicate that its gone.